pysat/instruments/__init__.py

```diff
@@ -11,6 +11,8 @@
            'demeter_iap', 'dmsp_ivm',
            'icon_ivm', 'icon_euv', 'icon_fuv', 'icon_mighti',
            'iss_fpmu',  'jro_isr', 'omni_hro', 'rocsat1_ivm',
+           'pysat_testing', 'pysat_testing_xarray', 'pysat_testing2d',
+           'pysat_testing2d_xarray', 'pysat_testmodel',
```

Comment:
Perhaps init could differentiate by having this in another list that is extended to the end of `__all__`?  It would be another (possibly unnecessary) step, but would mark these apart as different.

Reply:
Maybe?  Ultimately, these will be the only instruments in pysat (with 3.0), so I don't know if this is necessary.

Reply:
Oh, it isn't necessary.  It would visually separate them as we go forward in the future, but this is perfectly fine as is.